### ISSUE TITLE
[QUESTION] How to use custom prop / attribute name for classes?

### ISSUE THREAD

**Issue by KonradLinkowski** [francoismassart/eslint-plugin-tailwindcss#169]
I'm trying to disallow custom classes for [react-day-picker](https://react-day-picker.js.org/)

The code looks more or less like that:

Classes:
```
const classNames: ClassNames = {
  button_reset: 'custom-class', // this should fail lint check
  button: 'hover:bg-mono-20 cursor-pointer'
};
```
The component:
```
<DayPicker
  classNames={classNames}
  initialFocus={false}
  mode="single"
/>
 ```
 
 Tailwind generates classes just fine for that setup, but eslint ignores it.
 
 Is there any way to disallow custom class names in a setup like that, when a custom property name is used for `className`?

**Comment by francoismassart** [comment]
Hi @KonradLinkowski,
you could simply:

1. Add ctl to your project `npm i @netlify/classnames-template-literals`
2. Then update your code as such
```
import ctl from '@netlify/classnames-template-literals'
...
const classNames: ClassNames = {
  button_reset: ctl('custom-class'), // this will fail lint check
  button: ctl('hover:bg-mono-20 cursor-pointer') // this will be linted as well
};
```

By doing so, the argument of the call to [`ctl`](https://github.com/netlify/classnames-template-literals) will go through the rules of eslint-plugin-tailwindcss.

I you don't want to use `ctl`, just make your own function and set it up in the `callees` settings.

**Comment by francoismassart** [comment]
Let me know how it works for you 👍

**Comment by charkour** [comment]
We use a simple function to lint our className strings. 

Theoretically, in your build step, you could remove the callee.

```ts
export const c = (...strings: string[]) => strings;
```
